template.yml

```diff
@@ -97,6 +97,8 @@ Resources:
           ISSUER: !Ref Issuer
           OVERLAP_DURATION: !Ref OverlapDuration
           SECRETS_MANAGER_ENDPOINT: !Ref Endpoint
+          CLIENT_SECRET_KEY: secret
```

Comment:
I think that these environment variable values are unconfigurable without a template parameter, right?

Reply:
I'm not totally sure - I wasn't actually able to test this part of it out. I added new parameters and the !Ref like the others are set up. Should be better now